src/beheer-footer/beheer-footer.scss

```diff
@@ -1,12 +1,11 @@
-// this var is needed to build, but will be overridden by the app that implements the component
-$static-path: 'static';
+$pyoes-img-path: './img' !default;
```

Comment:
waarom deze var er nog in laten als ge toch image mee shipped?

Reply:
@roefem vond dat handig zodat we eventueel dat nog kunnen overschrijven in de parent app

Mij is het eender of we die var hier nog hebben of niet

Reply:
Kweet wel niet 100% zeker of dieje override wel werkt. Ffkes testen

Reply:
om te overriden-> in app.scss
```scss
$pyoes-img-path: "https://flyclipart.com/thumb2/kfc-new-logo-png-transparent-background-download-905143.png";

@import "oerelia/src/beheer-header/beheer-header";
```
![image](https://user-images.githubusercontent.com/62388265/182801512-7283bda7-037a-4330-84c1-944ab9a7df98.png)

Reply:
ik denk dat we hier 2 denwijzen door elkaar aan het zwieren zijn:
- ofwel zetten we de image in de parent app
- ofwel "hardcoderen" we ze hier 
ik zie geen enkel geval dat we de bestaande image ooit in 1 app anders zouden willen doen. 
kan op zich geen kwaad, maar er is het "gevaar" dat we in de toekomst niet meer weten welke image nu waar gebruikt  wordt, en dan dingen onnodig gaan override.

Reply:
![image](https://user-images.githubusercontent.com/5746523/182812765-cefdf0dc-d124-4065-b2be-56b4b2cfd5fd.png)

Reply:
> ik denk dat we hier 2 denkwijzen door elkaar aan het zwieren zijn:
> 
> * ofwel zetten we de image in de parent app
> * ofwel "hardcoderen" we ze hier
>   ik zie geen enkel geval dat we de bestaande image ooit in 1 app anders zouden willen doen.
>   kan op zich geen kwaad, maar er is het "gevaar" dat we in de toekomst niet meer weten welke image nu waar gebruikt  wordt, en dan dingen onnodig gaan override.

Kzie er op zich geen graten in om allebei te gebruiken, maar aangezien we toch overal al pyoes gebruiken zullen we die lijn doortrekken